### PR TITLE
Add support for updating configurations

### PR DESCRIPTION
**Pull request by amock**
Also update version to 0.5.7

### CODE REVIEW

File: ambariclient/models.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def stop(self):
 
     def restart(self):
         """Restarts this component on its host, if already installed and started."""
-        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN'):
+        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN', 'INSTALLED'):
```
Comment (jimbobhickville) [inline ambariclient/models.py]: I take it this works ok now then?  It used to complain when you tried to restart something that hadn't been previously started.

Comment (jimbobhickville) [inline ambariclient/models.py]: I think that about covers all the possible states now, so maybe we can just remove the check entirely and let the API tell them if they did something dumb?

Comment (amock) [inline ambariclient/models.py]: It does work now and removing it seems like a good idea, so I've done that.

**Comment by jimbobhickville** [comment]
+1

**Pull request merged**